setup.py

```diff
@@ -35,6 +35,7 @@
         'Programming Language :: Python :: 3.7',
         'Programming Language :: Python :: 3.8',
         'Programming Language :: Python :: 3.9',
+        'Programming Language :: Python :: 3.10',
```

Comment:
Not really related, but is this intentional?

Reply:
Yup

Reply:
Alrighty :D